Source/Core/Core/HW/EXI/EXI.cpp

```diff
@@ -62,6 +62,21 @@ void AddMemoryCards(int i)
 
   g_Channels[i]->AddDevice(memorycard_device, 0);
 }
+
+void AddSP1Device()
+{
+  TEXIDevices sp1_device;
```

Comment:
```suggestion
  TEXIDevices sp1_device = EXIDEVICE_BASEBOARD;
```

Then you only need to handle the else case instead of both the if and else.